tests/test_token_cache.py

```diff
@@ -28,7 +28,8 @@ def build_id_token(
     @staticmethod
     def build_response(  # simulate a response from AAD
             uid=None, utid=None,  # If present, they will form client_info
-            access_token=None, expires_in=3600, token_type="some type",
+            access_token=None, expires_in=3600, 
+            token_type="some type",
```

Comment:
Nit: If there is no actual change, we better keep existing code as-is, that way we will have an even cleaner PR to review. :-)